src/http/server.clj

```diff
@@ -293,6 +294,24 @@
          (log/infof "\"%s %s HTTP/1.1\" %s %s" method'  uri status len))
        response))))
 
+(defn pw-prompt []
+  (str (do (println "Password: ") (.readPassword (System/console)))))
+
+(defn maybe-inject-auth
+  [auth handler]
+  (fn [req]
+    (handler (if auth
```

Comment:
`(if (some? auth) ...`

Reply:
Previously `auth` could never be `false`, only `nil` or  bound to a string, so I don't think there is any semantic difference - I assume this is for better readability?   Now that `--auth` is required to have arguments (but may not have password),  I can probably eliminate the check entirely.